code/game/machinery/atmo_control.dm

```diff
@@ -545,6 +546,11 @@
 
 	add_fingerprint(usr)
 
+	if(href_list["in_set_rate"])
+		var/response=input(usr,"Set new flow rate, in L/s. \[0-[200]\]") as num
```

Comment:
I think you want have MAX_TRANSFER_RATE here as wel, instead of hardcoding 200?

Reply:
You're absolutely right, and I fixed it.